e2e/storybook/src/storybook-nested.test.ts

```diff
@@ -79,7 +79,7 @@ describe('Storybook generators and executors for standalone workspaces - using R
       writeFileSync(
         tmpProjPath(`src/app/test-button.tsx`),
         `
-          import { MyLib } from '@${appName}/my-lib';
+          import { MyLib } from 'my-lib';
```

Comment:
Since the standalone app isn't scoped by default now (just the app's name in `package.json`), the library does not default to a scope either.